python/hail/context.py

```diff
@@ -44,6 +47,16 @@ class HailContext(object):
     :vartype sc: :class:`.pyspark.SparkContext`
     """
 
+    @typecheck(sc=nullable(SparkContext),
+               appName=strlike,
+               master=nullable(strlike),
+               local=strlike,
+               log=strlike,
+               quiet=only(bool),
```

Comment:
I think only shouldn't be necessary here.  Actually, you set `tc = only(tc)` below, so just bool should work.